hh_aidoc[ESX]/client.lua

```diff
@@ -122,19 +122,43 @@ function DoctorNPC()
 	end
 
 	TaskPlayAnim(test1, "mini@cpr@char_a@cpr_str","cpr_pumpchest",1.0, 1.0, -1, 9, 1.0, 0, 0, 0)
-	exports['progressBars']:startUI(Config.ReviveTime, "The doctor is giving you medical aid")
+	TriggerEvent("mythic_progbar:client:progress", {
+        name = "unique_action_name",
+        duration = 10000,
+        label = "Giving CPR.",
+        useWhileDead = true,
+        canCancel = false,
+        controlDisables = {
+            disableMovement = true,
+            disableCarMovement = true,
+            disableMouse = false,
+            disableCombat = true,
+        }
+    }, function(status)
+        if not status then
+            -- Do Something If Event Wasn't Cancelled
+        end
+    end)
 	Wait(Config.ReviveTime)
 	ClearPedTasks(test1)
 	Citizen.Wait(500)
-	TriggerEvent('esx_ambulancejob:revive')
-	StopScreenEffect('DeathFailOut')	
-	Notify("Your treatment is done, you were charged: "..Config.Price)
-	RemovePedElegantly(test1)
-	DeleteEntity(test)
+	TriggerEvent('esx_ambulancejob:revive', GetPlayerServerId(PlayerId()))
```

Comment:
```suggestion
	TriggerEvent('esx_ambulancejob:revive', GetPlayerServerId(PlayerId()))
```
No need for the extra argument.

Reply:
hey bro can i get your discord id, i need a help of you

Reply:
Bombay#9032